app/_src/production/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: About Kuma in Production 
+content_type: explanation
+subtitle: Deploying Kuma in a production environment. 
+---
+
+After you've completed your initial test and assessment of {{site.mesh_product_name}}, it's time to deploy {{site.mesh_product_name}} in your production environment. Before doing so, it's important to understand the overall steps in the deployment process as well as the different options you have when deploying {{site.mesh_product_name}}. This guide walks you through the big picture steps and options so you can make the right decisions when it comes to deployment time.
+
+## Overview of deployment steps
+
+Deploying {{site.mesh_product_name}} to a production environment involves the following steps:
+
+1. [Decide which deployment topology you plan to use](#deployment-topologies).
+1. [Install `kumactl`](#kumactl).
+1. [Deploy the control plane](#control-plane-and-data-plane-architecture).
+1. [Configure the data plane](#control-plane-and-data-plane-architecture).
+1. [Configure security features for {{site.mesh_product_name}}](#{{site.mesh_product_name}}-security).
```

Comment:
```suggestion
1. [Configure security features for {{site.mesh_product_name}}](#kuma-security).
```
Didn't work in the link since the name is uppercase, my bad.